vividus-plugin-browserstack/src/main/resources/spring.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans https://www.springframework.org/schema/beans/spring-beans.xsd"
+       profile="browserstack">
+
+    <bean class="org.vividus.selenium.browserstack.BrowserStackCapabilitiesConfigurer" lazy-init="false">
```

Comment:
`lazy-init="false"` - why is it needed?

Reply:
removed, misunderstood some concept about collection wiring